Shoko.Server/Repositories/Cached/AnimeEpisodeRepository.cs

```diff
@@ -56,8 +56,9 @@ public SVR_AnimeEpisode GetByAniDBEpisodeID(int epid)
     /// Get the AnimeEpisode 
     /// </summary>
     /// <param name="name">The filename of the anime to search for.</param>
+    /// <param name="size">The size of the file in bytes</param>
     /// <returns>the AnimeEpisode given the file information</returns>
-    public SVR_AnimeEpisode GetByFilename(string name)
+    public SVR_AnimeEpisode GetByFilename(string name, long? size = null)
```

Comment:
Consider renaming this, or adding an extra helper to make this clearer.

Reply:
Should I split it into 2 functions, `GetByFilename(string name)` and `GetByFilenameAndSize(string name, long size)`?